zerver/management/commands/export.py

```diff
@@ -217,3 +218,7 @@ def percent_callback(bytes_transferred: Any) -> None:
             consent_message_id=consent_message_id,
             export_as_active=True if options["deactivate_realm"] else None,
         )
+
+        RealmAuditLog.objects.create(
+            realm=realm, event_type=RealmAuditLog.REALM_EXPORTED, event_time=timezone_now()
+        )
```

Comment:
Hmm, why shouldn't this be a change inside `export_realm_wrapper`?

Reply:
We already create the required auditlog when performing export of public data via organization's settings panel. That auditlog is used in `deferred_work` queue processor (see `event["type"] == "realm_export":`). A call to `export_realm_wrapper` is made, creating auditlog inside `export_realm_wrapper` will lead to duplicate logs which is not required.

Code flow for quick reference:

`export_realm()` view in `zerver/views/realm_export.py` -> `event["type"] == "realm_export":` block in `zerver/worker/deferred_work`